### PR TITLE
docs(checkbox): fix code sample

### PR DESCRIPTION
**Pull request by appsforartists**

### CODE REVIEW

**Comment by sgomes** [comment]
Thanks, @appsforartists, nice catch!

I'm not entirely sure if this is the right fix, though, as looking at the demos it looks like the `div` with class `mdc-checkbox__mixedmark` should be inside the `div` with class `mdc-checkbox__background` (and if that's the case there are other examples to fix in this document). I'll leave it to @traviskaufman to review, as he's more familiar with checkbox code than me.

**Comment by appsforartists** [comment]
I used the indentation to deduce where it was meant to go.  I trust TK knows better than I. :smiley:

**Comment by appsforartists** [comment]
https://github.com/material-components/material-components-web/tree/master/packages/mdc-form-field is nicely indented and has the right number of tags, so I've updated to match it.

File: packages/mdc-checkbox/README.md [hunk 1]
```diff
@@ -52,6 +57,7 @@ easily position checkboxes and their labels.
         <path class="mdc-checkbox__checkmark__path" fill="none" stroke="white"
               d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
       </svg>
+    </div>
```
Comment (traviskaufman) [inline packages/mdc-checkbox/README.md]: I think this part needs to be updated as well with what you did for the first example.

**Comment by appsforartists** [comment]
Like that?

**Pull request merged**